client/src/components/History/HistoryPublishedList.vue

```diff
@@ -62,18 +62,6 @@ const updateFilter = (newVal, append = false) => {
     filterText.value = oldValue.trim();
 };
 
-const onTagClick = (tag) => {
-    if (filterText.value.includes("tag:" + tag.label)) {
-        updateFilter(filterText.value.replace("tag:" + tag.label, ""));
-    } else {
-        if (filterText.value.length === 0) {
-            updateFilter("tag:" + tag.label, true);
-        } else {
-            updateFilter(" tag:" + tag.label, true);
-        }
-    }
-};
```

Comment:
Did you decide that clicking tags shouldn't result in this filter updating? I based the workflow list filtering around this and it seemed like a good idea?

Reply:
@jmchilton Correct - but only because this feature was not working (see this bug ticket I created here). I believe @itisAliRH suggested that the issue has to do with a case-sensitive search on the back-end -so there might be a PR out for that already..?

Reply:
@hujambo-dunia, clicking on a regular history tag or a "community tag" in the Published Histories seems to work for me on dev and it is case-insensitive :thinking:

Reply:
@hujambo-dunia As I mentioned in the #14985 description, only annotation filtering is case-sensitive (fixes here #15897) and other filters work fine.

Reply:
Thanks very much @itisAliRH for the clarification.

Another question: understandably, the original backend tests started to fail after I updated to the new component. My ideas/changes were insufficient to fix the issue. Could you advise/help? I thought the issue was a failed CSS selector because the new component lacks a "class" parameter value, so I reference the DOM element using `CSS_SELECTOR` in Python. However, the Selenium tests `test_published_histories_tag_click` and `test_published_histories_search_*` are still failing and I do not understand the original intent behind those tests (what are they testing for?).